lib/gossip.js

```diff
@@ -191,14 +190,9 @@ async function _getNeeded({ledgerNode, needed, peerId}) {
             return resolve();
           }
           // last chunk
-          // the correct number of events have been received, submit the batch
+          // the correct number of events have been received
           if(events.length === needed.length) {
-            return _events.addBatch({events, ledgerNode, needed})
-              .then(resolve)
-              .catch(error => {
-                logger.error('An error occurred in `addBatch`', {error});
-                reject(error);
-              });
+            return resolve();
```

Comment:
Seems like this could be combined with the above conditional now -- they both do the same thing.